### ISSUE TITLE
Code style guide ... pep8 standard

### ISSUE THREAD

**Issue by coleshaw** [CLIxIndia-Dev/CLIxChat#2]
So this is really nit-picky, but it would help the readability of the code if you followed the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/). Depending on what editor you use, there should be various plugins that flag pep8 violations in the editor. Here's an example of what it would look like. Hovering over the red line tells me what the editor thinks could be improved.

<img width="588" alt="captura de pantalla 2017-09-15 a la s 3 20 05 pm" src="https://user-images.githubusercontent.com/4930129/30499647-6331c4e0-9a29-11e7-86a6-977727e26f91.png">

I've never actually read the entire pep8 guide, but in general:

* All imports at the top of the file, starting with standard library imports (`import X`) in alphabetical order, then third-party packages, then relative imports.
* Two new lines before any new function / class definition.
* Very picky about spacing and how things should line up.

I can help you find a pep8 plugin and configure it, if you let me know what editor you're using.

**Comment by brandonhanks** [comment]
Yeah I actually have it enabled and I can see the issues (I use pycharm). I'll clean it up though and push. Thanks!
